larray/core/constrained.py

```diff
@@ -327,7 +327,7 @@ class ConstrainedSession(Session, AbstractConstrainedSession, metaclass=ModelMet
         >>> # here the result of the multiplication of the 'population' array by the 'mortality_rate' array
         >>> # is automatically converted to an integer array (printing a warning message)
         >>> m.deaths = m.population * m.mortality_rate
-        >>> print(m.deaths.info)
+        >>> print(m.deaths.info)                            # doctest: +SKIP  (linux uses dtype=int64 instead of int32)
```

Comment:
wouldn't using np.int64 (or int32) explicitly when creating the array, instead of "int" solve the problem without having to skip the test?

Reply:
To not afraid our users. Lot of them have no idea what ``np.x`` means.